proto/topodata.proto

```diff
@@ -396,6 +399,13 @@ message ThrottlerConfig {
   map <string, double> metric_thresholds = 7;
 }
 
+// VtorcConfig stores configuration for Vtorc.
+message VtorcConfig {
```

Comment:
> message VtorcConfig

`OrchestrationConfig` is an alternative that doesn't jump to what component does it 🤔